pages/account/manage/link/[[...data]].js

```diff
@@ -30,6 +34,12 @@ export async function getServerSideProps(context) {
   const id = context.query.data ? context.query.data[0] : undefined;
 
   let link = {};
+  let groups = [];
```

Comment:
Maybe `linkGroups` is a beter name, than it's clear what kind of groups this variable holds